src/unix/sock_ctrl_msg.rs

```diff
@@ -635,18 +586,18 @@ mod tests {
     fn send_more_recv_less2() {
         let (s1, s2) = UnixDatagram::pair().expect("failed to create socket pair");
 
-        let evt1 = EventFd::new(0).expect("failed to create eventfd");
-        let evt2 = EventFd::new(0).expect("failed to create eventfd");
-        let evt3 = EventFd::new(0).expect("failed to create eventfd");
-        let evt4 = EventFd::new(0).expect("failed to create eventfd");
+        let (_, evt_notifier1) = pipe().expect("failed to create pipe");
+        let (_, evt_notifier2) = pipe().expect("failed to create pipefd");
+        let (_, evt_notifier3) = pipe().expect("failed to create pipefd");
+        let (_, evt_notifier4) = pipe().expect("failed to create pipefd");
```

Comment:
Ditto, please check it next time, I thought was obvious to fix in all places, no?